cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java

```diff
@@ -258,4 +320,21 @@ private void verifySearchResultContains(String url, List<String> expectedValues)
     Assert.assertTrue(reply.containsAll(expectedValues));
   }
 
+  class TimeSeriesResult {
```

Comment:
should be separate, anyhow, we'll clenaup with https://github.com/caskdata/cdap/pull/1308